web/app.php

```diff
@@ -5,7 +5,7 @@
 
 require_once __DIR__.'/../vendor/autoload.php';
 
-$app = new AppKernel('ci');
+$app = new AppKernel(getenv('ENVIRONMENT_NAME'));
```

Comment:
Needs a default of `dev`?

Reply:
Can't hurt, although it should be provided by `dev.env`